src/test/kotlin/no/elhub/auth/features/documents/DocumentValidationHelper.kt

```diff
@@ -70,4 +78,24 @@ object DocumentValidationHelper {
         signingCertificateDN shouldBe testCertificate.issuer.rfC2253
         signingCertificateSerialNumber shouldBe testCertificate.serialNumber.toString()
     }
+
+    fun validateDocumentIsCompliant(byteArray: ByteArray): Boolean {
```

Comment:
:art: Could presumable drop the wrapper object and simply have these as functions?